report/report.py

```diff
@@ -87,6 +93,12 @@ async def cmd_emergency(self, ctx: commands.Context, *, message: str = None):
             mod_pings = " ".join([i.mention for i in log.members if not i.bot])
         await log.send(content=mod_pings, embed=data)
 
+        report_reply = self.make_reporter_reply(ctx, message, True)
+        if ctx.channel.type == discord.ChannelType.private:
```

Comment:
This is a guild only command, so the channel type will never be private

Reply:
DM logic has been removed.